src/extension.node.ts

```diff
@@ -210,9 +214,28 @@ async function activateComponents(
     return activateLegacy(context, serviceManager, serviceContainer);
 }
 
+function addConsoleLogger() {
+    if (process.env.VSC_JUPYTER_FORCE_LOGGING) {
+        let label = undefined;
+        // In CI there's no need for the label.
+        const isCI = process.env.TF_BUILD !== undefined || process.env.GITHUB_ACTIONS === 'true';
```

Comment:
Aren't we already importing an "isCI" into this file?

Reply:
Thanks. Didn't notice that.